android/expoview/src/main/java/host/exp/exponent/experience/ReactNativeActivity.kt

```diff
@@ -614,6 +614,9 @@ abstract class ReactNativeActivity :
   val devSupportManager: RNObject?
     get() = reactInstanceManager.takeIf { it.isNotNull }?.callRecursive("getDevSupportManager")
 
+  val jsExecutorName: String?
+    get() = reactInstanceManager.takeIf { it.isNotNull }?.callRecursive("getJsExecutorName")?.get() as? String
```

Comment:
```suggestion
    get() = reactInstanceManager?.callRecursive("getJsExecutorName")?.get() as? String
```
That should do the same think

Reply:
the `reactInstanceManager` is not null, but a concrete instance of [RNObject wrapper](https://github.com/expo/expo/blob/main/android/expoview/src/main/java/host/exp/exponent/RNObject.kt), i think we still use its `isNotNull` method. does that make sense to you?